client/imports/app/new-forum/new-forum.html

```diff
@@ -0,0 +1,16 @@
+<aside>
```

Comment:
This module shouldn't be a replacement for the home-component so why does it contain the structure of the *HomeComponent*.